src/math/calculation.js

```diff
@@ -841,21 +841,20 @@ function hypot(x, y, z) {
  * function setup() {
  *   createCanvas(windowWidth, windowHeight);
  *   fill(0);
- *   let nums = [
- *     Infinity,
- *     1e-10,
- *     1.4215e-10,
- *     14213.214e-10,
- *     142144.42142241241212521e10,
- *     133232e10,
- *     1.2412,
- *     '241.12421'
- *   ];
- *   let spacing = 150;
- *   for (let i = 0; i < nums.length; i++) {
- *     text(String(nums[i]), i * spacing, 50);
- *     text(String(fract(nums[i])), i * spacing, 100);
- *   }
+ *   text(String(Infinity), 0, 50);
+ *   text(String(fract(Infinity)), 0, 100);
+ *   text(String(1e-10), 150, 50);
+ *   text(String(fract(1e-10)), 150, 100);
+ *   text(String(1.4215e-10), 300, 50);
+ *   text(String(fract(1.4215e-10)), 300, 100);
+ *   text(String(14213.214e-10), 450, 50);
+ *   text(String(fract(14213.214e-10)), 450, 100);
+ *   text(String(133232e10), 600, 50);
+ *   text(String(fract(133232e10)), 600, 100);
+ *   text(String(1.2412), 750, 50);
+ *   text(String(fract(1.2412)), 750, 100);
+ *   text(String('241.12421'), 900, 50);
+ *   text(String(fract('241.12421')), 900, 100);
```

Comment:
There's too many cases here. The examples are not meant to show all possible uses of the function so just one or two examples will be enough. Also is the `String()` conversion needed here?

Reply:
I used the `String()` conversion to show that the function also works for undefined input, but if all these cases are not needed I'll remove them as you asked me to.